README.md

```diff
@@ -21,7 +21,15 @@ Usage
   
         $ npm install -g osmtogeojson
         $ osmtogeojson file.osm > file.geojson
+
+  Command line for Flat property usage:
+  
+        $ osmtogeojson -e=true file.osm > file.geojson
+
+  The supported command line options are shown with
   
+        $ osmtogeojson --help
```

Comment:
Mentioning `--help` is a very good idea! I've applied the change manually in 6441adc75110c58af7b3932e41680363c160375f